templates/detail.jinja2

```diff
@@ -4,7 +4,7 @@
     <h2 class="title">{{ entry.title }}</h2>
     <hr>
     <p>
-        {{ entry.text }}
+        {{ entry.text | safe }}
```

Comment:
What if your entry object had a "markdown" method that returned the rendered markdown text?